dts/bindings/sensor/st,lsm6dsl-i2c.yaml

```diff
@@ -1,22 +1,22 @@
 #
-# Copyright (c) 2017, Linaro Limited
+# Copyright (c) 2018, Linaro Limited
```

Comment:
I'm not sure we should update Copyright year.
Do you have a reason for this update? We don't do this generally (maybe we're wrong though).

Reply:
I think you are right. I changed it because it is a new file, but actually it DOES NOT have a new content. I'll revert this line back to 2017

Reply:
Actually I slightly changed the content.
We have two options:

1. I leave just the original copyright
# Copyright (c) 2017, Linaro Limited

2. I add also a new STMicro copyright
# Copyright (c) 2017, Linaro Limited
# Copyright (c) 2018 STMicroelectronics

what do you think @erwango ?

Reply:
Since change is rather major vs the initial file content, I think option 2 is legit here